### ISSUE TITLE
Sometimes, NVDA 2022.1 fails to sellect

### ISSUE THREAD

**Issue by jacobchawala** [nvaccess/nvda#13713]
In Microsoft word 2021, NVDA sometimes fails to select. I was trying to select a word by pressing control+shift+right arrow, but it doesn't select a word and it does not even saying "selected." to curb the problem, I have to restart NVDA.

### Actual behavior:
Sometimes, NVDA fails to select
### Expected behavior:
NVDA has to be able to select characters, words, setences, paragraphs, page, and whole document all the time.
#### NVDA installed/portable/running from source:
NVDA installer
#### NVDA version:
NVDA 2022.1 RC2
#### Windows version:
Operating System: Windows 11 Pro 64-bit (10.0, Build 22621) (22621.ni_release.220506-1250)
#### Name and version of other software in use when reproducing the issue:
Microsoft Office 2021 professional plus
#### Other information about your system:
Machine name: JACOBCHAWALA
Machine Id: {8A4A0A4B-6524-4A4E-9CDB-22AD55F2EB94}
Language: English (Regional Setting: English)
System Manufacturer: HP
System Model: HP ENVY x360 Convertible 15-cn0xxx
BIOS: F.28 (type: UEFI)
Processor: Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (8 CPUs), ~2.0GHz
Memory: 16384MB RAM
Available OS Memory: 16212MB RAM

#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
It also occurs in NVDA 2022.1 RC1
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by seanbudd** [comment]
Welcome @jacobchawala ,

Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences.
Could you also confirm that this isn't an issue in 2021.3.5?

Kind Regards,
NV Access Software Developers

**Comment by jacobchawala** [comment]
> Welcome @jacobchawala ,
> 
> Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences. Could you also confirm that this isn't an issue in 2021.3.5?
> 
> Kind Regards, NV Access Software Developers

It is difficult to provide the log file of this issue since it does not occur always. but I confirm that this isn't an issue in 2021.3.5.

**Comment by jacobchawala** [comment]
> > Welcome @jacobchawala ,
> > Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences. Could you also confirm that this isn't an issue in 2021.3.5?
> > Kind Regards, NV Access Software Developers
> 
> It is difficult to provide the log file of this issue since it does not occur always. but I confirm that this isn't an issue in 2021.3.5.

Eventually, I have managed to upload two NVDA log files! you can check them.

**Comment by seanbudd** [comment]
Which version of office are you using? Can you provide the version number and build number?
https://support.microsoft.com/en-us/office/about-office-what-version-of-office-am-i-using-932788b8-a3ce-44bf-bb09-e334518b8b19

Have you tried updating office?

**Comment by seanbudd** [comment]
Ignore my previous message, I see you have updated #13712 with your office version number

**Comment by seanbudd** [comment]
Does updating word fix this issue?

**Comment by jacobchawala** [comment]
updating word did not fix this issue.

**Comment by Adriani90** [comment]
@jacobchawala can you still reproduce this issue with NVDA 2023.2?

**Comment by Adriani90** [comment]
We didn't get any updates from the initial author since more than 1 year and I cannot reproduce this issue. Closing as abandoned. If you are still having this issue, please comment and we can reopen.
